src/api/spec/factories/kiwi_image.rb

```diff
@@ -0,0 +1,28 @@
+FactoryGirl.define do
+  factory :kiwi_image, class: Kiwi::Image do
+    name { Faker::Name.name }
+    md5_last_revision nil
+
+    factory :kiwi_image_with_package do
+      transient do
+        package_name 'package_with_kiwi_image'
+        with_kiwi_file false
+        kiwi_file_name { "#{package_name}.kiwi" }
+        file_content { Kiwi::Image::DEFAULT_KIWI_BODY }
+        project nil
```

Comment:
Mmmm.. no I guess, if you don put this line is not declared as a transient param and whenever you use this it will try to assign it to a Kiwi::Image model...